baseplate/core.py

```diff
@@ -592,8 +592,8 @@ def configure_tracing(self, tracing_client, *args, **kwargs):
         if not isinstance(tracing_client, TracingClient):
             warn_deprecated("Passing tracing configuration directly to "
                             "configure_tracing is deprecated in favor of "
-                            "using baseplate.make_tracing_client and passing "
-                            "the constructed client on.")
+                            "using baseplate.tracing_client_from_config and "
```

Comment:
Good change, but this is unrelated to the E501 thing. Can it be split out into its own commit please?